integration-templates/zoho-mail/zoho-mail-emails.ts

```diff
@@ -0,0 +1,66 @@
+import type { NangoSync, ZohoMailEmail } from './models';
+
+const LIMIT = 100;
+
+export default async function fetchData(nango: NangoSync) {
+    let totalRecords = 0;
+    let offset = 1;
+
+    const metadata = (await nango.getMetadata()) || {};
+    const accountId = metadata['accountId'] ? String(metadata['accountId']) : '';
+
```

Comment:
Shouldn't this sync bail if there is no `accountId`?

Reply:
Is the accountId not apart of the nango metadata? Where does the accountId come from?

Reply:
@khaliqgant Yes, the sync should fail if there is no accountId. I have updated my pull request. Thanks.